website_demo/index.html

```diff
@@ -0,0 +1,326 @@
+        <link rel="manifest" href="manifest.json">
+        <link rel="mask-icon" href="safari-pinned-tab.svg" color="#5bbad5">
+        <link rel="stylesheet" href="css/normalize.min.css">
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+        <link rel="stylesheet" href="css/jquery.fancybox.css">
+        <link rel="stylesheet" href="css/flexslider.css">
+        <link rel="stylesheet" href="css/styles.min.css">
+        <link rel="stylesheet" href="css/queries.css">
+        <link rel="stylesheet" href="node_modules/animate.css/animate.min.css">
+        <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css">
+        <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+    </head>
+
+    <body id="top">
+
+        <section class="hero">
+            <section class="navigation">
+                <header>
+                    <div class="header-content">
+                        <div class="logo"><a href="#"><img src="img/hcp_logo_64.png" alt="Logo"></a></div>
+                        <div class="header-nav">
+                            <nav>
+                                <ul class="primary-nav">
+                                    <li><a href="#intro">About Us</a></li>
+                                    <li><a href="#events">Demo</a></li>
+                                    <li><a href="#instagram"></a></li>
+                                </ul>
+                                <ul class="member-actions">
+                                    <!-- <li><a href="#map" class="login">Venue</a></li> -->
+                                    <!-- <li><a href="#rsvp" class="btn-white btn-small">RSVP</a></li> -->
+                                </ul>
+                            </nav>
+                        </div>
+                        <div class="navicon">
+                            <a class="nav-toggle" href="#"><span></span></a>
+                        </div>
+                    </div>
+                </header>
+            </section>
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-10 col-md-offset-1">
+                        <div class="hero-content text-center">
+                            <img src="img/HCPLogoText-Crop.png">
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-sm-4 hidden-xs"></div>
+                    <div class="col-sm-4 hidden-xs text-center"><a href="#rsvp"
+                            class="btn btn-accent btn-large rsvp-btn">Join our mailing list</a></div>
+                    <div class="col-sm-4 hidden-xs"></div>
+                </div>
+            </div>
+            <div class="down-arrow floating-arrow"><a href="#invitation"><i class="fa fa-angle-down"></i></a></div>
+        </section>
+
+        <section id="invitation" class="section-padding">
```

Comment:
Could you remove unused sections? I believe `invitation`, `instagram`, `eng-pics`, `video-bg` are unused. We can add back / create new sections when needed.

Reply:
I was thinking about leaving Instagram to leave pictures of our product and potentially use video-bg as an add-on. But I removed the other two that were not being used